test/run_test.py

```diff
@@ -163,6 +163,19 @@ def test_distributed(python, test_module, test_directory, options):
 }
 
 
+def parse_test_module(test):
+    idx = test.find('.')
```

Comment:
I think this function can be replaced with `test.split('.')[0]`, since it will return the whole string when there is not dot

Reply:
You are right. Will do it.